app/liquid_tags/forem_tag.rb

```diff
@@ -1,5 +1,5 @@
 module ForemTag
-  REGISTRY_REGEXP = %r{#{URL.url}/?}
+  REGISTRY_REGEXP = %r{#{URL.url.gsub("\.", "\\.")}/\b([\w-]+)?}
```

Comment:
What would a spec look like to test the adjusted regex?

Reply:
filling in jeremyf's answer for him:

the new regex pattern matches a _different_ set of inputs than the original. We should provide some example input that's now accepted and wouldn't have been.

Reply:
I think we mean "that's not accepted and would have been". Ok.